common-features/document-processing/pdf.md

```diff
@@ -0,0 +1,58 @@
+            
+__RadPdfProcessing__ is a processing library that allows to create, import and export PDF documents.
+
+![RadPdfProcessing Overview image](https://docs.telerik.com/devtools/document-processing/libraries/radpdfprocessing/images/RadPdfProcessing_Overview_WPF_01.PNG)
+
+The API of RadPdfProcessing contains two different editors, enabling you to choose between editing in a flow-like manner, or using the much more powerful and flexible fixed document structure and drawing on the page.
+            
+The document model of the library provides support for:
+
+* [**Pages**](https://docs.telerik.com/devtools/document-processing/libraries/radpdfprocessing/model/radfixedpage): Adding, modifying or removing of pages in a document. The properties enable you to change the size of the page, its rotation and more.
+
+* **Automatic layout**: Although the PDF format is fixed, sometimes you will need to insert the content in a way that flows on the page. RadPdfProcessing enables to achieve this easily by using  [blocks](https://docs.telerik.com/devtools/document-processing/libraries/radpdfprocessing/editing/block), [tables](https://docs.telerik.com/devtools/document-processing/libraries/radpdfprocessing/editing/table) and [lists](https://docs.telerik.com/devtools/document-processing/libraries/radpdfprocessing/editing/list).
+
+* [**Images**](https://docs.telerik.com/devtools/document-processing/libraries/radpdfprocessing/model/image): Decoded on demand to achieve better performance. The API enables you to obtain the encoded image data. You can also control the image quality when saving the document.
+
+* [**Geometries**](https://docs.telerik.com/devtools/document-processing/libraries/radpdfprocessing/concepts/geometry): Enable you to describe the geometry of a 2D shape.
+
+* [**Form XObjects**](https://docs.telerik.com/devtools/document-processing/libraries/radpdfprocessing/model/form): The Form XObjects enables you to describe composite objects (consisting of text, images, vector elements, etc.) within a PDF file and reuse this content among the document, for smaller document size and better rendering performance.
+
+* [**Interactive Forms**](https://docs.telerik.com/devtools/document-processing/libraries/radpdfprocessing/model/interactive-forms/overview): Create and modify PDF files containing textboxes, buttons, listboxes and other interactive controls making available for the PDF file user to interactively fill some data in the PDF document and/or digitally sign the filled document.
+
+* [**Digital Signature**](https://docs.telerik.com/devtools/document-processing/libraries/radpdfprocessing/features/digital-signature): The digital signature feature enables you to sign and validate PDF document.
```

Comment:
This is currently not supported in the Net Standard version, so we can remove it from here.